guides/source/layouts_and_rendering.md

```diff
@@ -417,6 +418,44 @@ render formats: [:json, :xml]
 
 If a template with the specified format does not exist an `ActionView::MissingTemplate` error is raised.
 
+##### The `:variants` Option
+
+This tells rails to look for template variations of the same format.
```

Comment:
rails should be Rails as that's what is used in this file and the project in general as a practice.

Reply:
Ahh good catch. We probably should have a [danger/peril](http://danger.systems) bot setup on this to do auto-reviews for things like this.

I'll submit a PR w/ that fix.